### PR TITLE
docs: update deprecated/outdated compose syntax and filenames

### PR DESCRIPTION
**Pull request by dvdksn**
Not an exhaustive update (didn't want to touch the `stack.yml` stuff yet, could do in a follow-up) but updated references to old `docker-compose` binary and changed the filename from `docker-compose.yml` to `compose.yaml`, where it made sense to do so.

### CODE REVIEW

**Review by yosifkit — APPROVED** [review]

**Comment by tianon** [comment]
We really went with the four-character file extension convention? :sob: (`compose.yaml`)

File: convertigo/content.md [hunk 1]
```diff
@@ -275,13 +275,13 @@ The default `ENABLE_JDWP_DEBUG` value is **false** and can be defined this way:
...
-## Pre configurated Docker compose stack
+## Pre-configurated Docker Compose file
 
-You can use this [stack](https://github.com/convertigo/docker/blob/master/compose/mbaas/docker-compose.yml) to run a complete Convertigo Low Code server with FullSync repository and MySQL analytics in a few command lines.
+You can use [this Docker Compose file](https://github.com/convertigo/docker/blob/master/compose/mbaas/docker-compose.yml) to run a complete Convertigo Low Code server with FullSync repository and MySQL analytics in a few command lines.
 
 ```console
 $ mkdir c8oMBaaS
 $ cd c8oMBaaS
 $ wget https://raw.githubusercontent.com/convertigo/docker/master/compose/mbaas/docker-compose.yml
```
Comment (tianon) [inline convertigo/content.md:285]: Does this need `-O compose.yaml`?

Comment (tianon) [reply convertigo/content.md:285]: I guess not because Compose will still pick up the older filename? :eyes:

**Review by tianon — APPROVED** [review]
Generally looks OK, thanks!

I _really_ dislike `compose.yaml` and would prefer `compose.yml` (or even `docker-compose.yml` :upside_down_face:) but I'll defer to the canonical upstream convention preference. :bow:

**Comment by tianon** [comment]
Took a bunch of digging, but finally found what I was looking for in https://github.com/compose-spec/compose-go/blob/9a9cc5d9c3453326bb8d506259bcfccd68af110b/cli/options.go#L384-L385

```go
// DefaultFileNames defines the Compose file names for auto-discovery (in order of preference)
var DefaultFileNames = []string{"compose.yaml", "compose.yml", "docker-compose.yml", "docker-compose.yaml"}
```

So, I guess this _is_ the upstream canonical preference. :upside_down_face:

**Pull request merged**